### ISSUE TITLE
False positive with UselessParentheses and ** operator

### ISSUE THREAD

**Issue by mattiabasone** [slevomat/coding-standard#766]
I'm using EasyCodingStandard with PHP 7.3 and when I run the inspection it reports the following issue, but removing the brackets the result will change.

```
-            $devs_red[] = ($all_red[$i] - $mean_red) ** 2;
-            $devs_green[] = ($all_green[$i] - $mean_green) ** 2;
-            $devs_blue[] = ($all_blue[$i] - $mean_blue) ** 2;
+            $devs_red[] = $all_red[$i] - $mean_red ** 2;
+            $devs_green[] = $all_green[$i] - $mean_green ** 2;
+            $devs_blue[] = $all_blue[$i] - $mean_blue ** 2;

Applied checkers:

 * SlevomatCodingStandard\Sniffs\PHP\UselessParenthesesSniff.UselessParentheses
```

**Comment by mmoll** [comment]
GH-730?

**Comment by mattiabasone** [comment]
> [GH-730](https://github.com/slevomat/coding-standard/pull/730)?

Yep!

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/244e6d3f89836fa0e60c7c2fadb4b639c8491f0e
